pack/fhv/fhvc.json

```diff
@@ -90,9 +90,407 @@
+    "double_sided": true,
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "flavor": "Lizzie's glowing footprints lead to an unexplored chamber deep in the caverns. Through your exploration, you now know how to lure the ravenous creatures with glowing moss out of the darkness and into the light.",
+    "illustrator": "Drazenka Kimpel",
+    "name": "On the Trail",
+    "pack_code": "fhvc",
+    "position": 32,
+    "quantity": 1,
+    "stage": 2,
+    "text": "[action] Spend 1[per_investigator] clues and choose an enemy at a connecting [[Dark]] location: Move that enemy to your location./n<b>Objective</b> - When an investigator enters Fungal Cave, immediately advance.",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "The towering abomination thrashes and squirms, its massive bulk shuddering in violent death throes. At long last, the monstrosity goes still. A filmy blue fluid leaks from its eyes and carapace, smelling like more awful, rotten shellfish. Covering your nose and mouth, you head back to Akwan.",
+    "back_name": "The Bigger They Are...",
+    "back_text": "<b>(→R1)</b> <i>(page 26)</i>.",
+    "clues": null,
+    "code": "10533",
+    "double_sided": true,
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 6,
+    "faction_code": "mythos",
+    "flavor": "The abomination from the depths of the save stalks you through the tunnels. You can only hope you can buy Helen and her sister enough time to escape.",
+    "illustrator": "Drazenka Kimpel",
+    "name": "Face to Carapace",
+    "pack_code": "fhvc",
+    "position": 33,
+    "quantity": 1,
+    "stage": 3,
+    "text": "[action] Spend 1[per_investigator] clues and choose an enemy at a connecting [[Dark]] location: Move that enemy to your location./n<b>Objective</b> - If Limulus Hybrid is defeated, immediately advance.",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "The fishing village of Akwan sits quietly in a rocky bay on the northern shores of Hemlock Isle. A few people mill around in the village's boardwalks.",
+    "clues": 0,
+    "code": "10534",
+    "double_sided": true,
+    "encounter_code": "the_lost_sister",
+    "encounter_position": 7,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Akwan",
+    "pack_code": "fhvc",
+    "position": 34,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "[action] <b>Resign.</b> You give up the search.\nIf it is <b>Day 1</b> or <b>Day 2</b>, Akwan gains \"[action] <b>Parley.</b> You ask around the village about the coastal caves. Look at the top 3 cards of the Caverns deck. Return them to the top of the Caverns deck in any order.\"",
+    "traits": "Coastal.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "After more aeons of descent I saw some side passages or burrows leading from unknown recesses of blackness to this shaft of nighted mystery\n<cite>H.P. Lovecraft, \"The Festival\"</cite>",
```

Comment:
```suggestion
    "back_flavor": "After more aeons of descent I saw some side passages or burrows leading from unknown recesses of blackness to this shaft of nighted mystery\n - H.P. Lovecraft, \"The Festival\"",
```

Reply:
I think FHV and TSK have used <cite> but it's not what we used to use, so I'm going to remove it throughout.